### PR TITLE
Fix loc after file resolution

### PR DESCRIPTION
**Pull request by gares**
@FissoreD would you mind testing this version of Elpi? It should give (in coq-elpi) better file names in error messages

### CODE REVIEW

**Comment by FissoreD** [comment]
Cannot find the branch:
image

**Comment by FissoreD** [comment]
What command should I write to get it, I did git remote update, but still not found

**Comment by gares** [comment]
`opam pin add elpi https://gihub.com/LPCIC/elpi.git#fix-loc-after-file-resolution`

**Comment by gares** [comment]
I like the approach, **** dune!

**Comment by gares** [comment]
elpi already depends on `Re`, and `open Re` gives you a `Str` module.
you should remove the `str` dependency and just open `Re` where needed.

File: src/parser/parse.ml [hunk 1]
```diff
@@ -67,6 +67,7 @@ let already_parsed = Hashtbl.create 11
 let () =
   parse_ref := (fun ?cwd filename ->
   let filename = C.resolver ?cwd ~unit:filename () in
+  let dest = Re.Str.replace_first (Re.Str.regexp "/_build/default") "" filename in
```
Comment (gares) [inline src/parser/parse.ml:70]:
```suggestion
  let dest = Re.Str.replace_first (Re.Str.regexp "/_build/[^/]+") "" filename in
```

**Pull request merged**